src/docs/self-hosted/logs.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: "Logs rotation"
+---
+
+By default docker writes logs of a container to one log file which can be giant - tens of gigabytes.
+
+To reduce a used space for logs you can 
+ * modify docker-compose.yaml file by adding into a common section for each container following logging settings:
```

Comment:
We use `.yml` extension in self-hosted, right now, I'm changing it just for convenience.
```suggestion
 * modify docker-compose.yml file and add following logging settings into a common section for each container:
```

Reply:
Please see next comment